Sources/PitchSpeller/Data Structures/Graph.swift

```diff
@@ -29,11 +29,15 @@ public struct Graph <Value: Hashable>: Hashable {
 
         public let source: Node
         public let destination: Node
-        public var value: Double
+        public var value: EdgeCapacity
```

Comment:
Instead of directly changing the concrete type of `Edge.value`, I'd suggest making it generic (over [`Numeric`](https://developer.apple.com/documentation/swift/numeric), etc.). Otherwise, we are tying this `Graph` implementation specifically to this use case.

Reply:
cool I'll look into Numeric - seems like a good choice